### PR TITLE
[LIBXSMM] Add libxsmm to tvm ci

### PR DESCRIPTION
**Pull request by zhuwenxi**
RFC: https://github.com/apache/tvm-rfcs/pull/47

This is the first PR in the integration plan.

### CODE REVIEW

**Comment by zhuwenxi** [comment]
@comaniac Could you help review this PR?

File: docker/install/ubuntu_install_libxsmm.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+set -e
+set -u
+set -o pipefail
+
+pushd /usr/local/
+wget -q https://github.com/libxsmm/libxsmm/archive/refs/tags/1.17.tar.gz
+tar -xzf 1.17.tar.gz
+pushd ./libxsmm-1.17/
+make STATIC=0 -j10
```
Comment (comaniac) [inline docker/install/ubuntu_install_libxsmm.sh:27]: better to config the thread number in a more flexible way.
```suggestion
make STATIC=0 -j$(($(nproc) - 1))
```

**Review by comaniac — APPROVED** [review]

**Comment by masahi** [comment]
@zhuwenxi please run another job. I can take care of the image update. Note that changing the docker script alone doesn't update the CI environment.

**Comment by zhuwenxi** [comment]
> @zhuwenxi please run another job. I can take care of the image update. Note that changing the docker script alone doesn't update the CI environment.

Thank you @masahi for the notice that changing docker script alone is not enough. I'm wondering is there any PR I can refer to fully update the CI?

**Comment by masahi** [comment]
The actual update needs to be done by a committer. I can do that this week.

**Comment by masahi** [comment]
The CI image has been updated by https://github.com/apache/tvm/pull/10889. We are ready to use libxsmm on CI.

**Pull request merged**